README.adoc

```diff
@@ -37,6 +37,8 @@ WARNING: We recommend using Slim 2.1.0 until https://github.com/asciidoctor/asci
 
   $ git clone git://github.com/hakimel/reveal.js.git
 
+WARNING: Currently asciidoctor-reveal.js is not compatible with reveal.js 3.0.0, be careful to clone or copy version 2.6.2 or older
```

Comment:
Perhaps we should recommend cloning a specific branch in the `git clone` statement. wdyt?

    $ git clone -b 2.6.2 https://github.com/hakimel/reveal.js

Notice I also updated the clone URL to be https as I've heard this is much better for anyone behind a proxy.

We should also start thinking about recommending the use of Bower or another tool to get reveal.js rather than just git clone, but that's a topic for a separate issue.